float64 distance_m

string FLAG_SAFE=safe
string FLAG_UNSAFE=unsafe
string FLAG_CRASH=crash
string _FLAG_ERROR_=SRV_ERROR
string flag